test/e2e/e2e_tests_suite_test.go

```diff
@@ -73,3 +75,12 @@ var _ = BeforeSuite(func(done Done) {
 
 	close(done)
 }, 60)
+
+func hugepageOrSkip() {
```

Comment:
Nit.
hugepageOrSkip -> HugePageOrSkip

To keep naming consistent with other functions in the file.

Reply:
From my point of view, the name of that function is correct. I don't want to export it outside e2e package and due to that it starts from lower case.

Reply:
Oh yes, you're right.